__tests__/issues.ts

```diff
@@ -94,3 +94,33 @@ describe('Issue #1293', () => {
     expect(secondState).toEqual(firstState);
   });
 });
+
+describe('Issue #1643', () => {
```

Comment:
are the issue numbers on the main repo the same? might be hard to track otherwise

Reply:
The two issues your pointed are in fact really the issues in immutable-js repository !